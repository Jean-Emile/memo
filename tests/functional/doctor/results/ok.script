{
    "drives": [
        {
            "name": "bob/aws_drive",
            "sane": true,
            "warning": false
        },
        {
            "name": "bob/fs_drive",
            "sane": true,
            "warning": false
        }
    ],
    "leftovers": [],
    "networks": [
        {
            "linked": true,
            "name": "bob/aws_network",
            "sane": true,
            "warning": false
        },
        {
            "linked": true,
            "name": "bob/fs_network",
            "sane": true,
            "warning": false
        }
    ],
    "sane": true,
    "storage resources": [
        {
            "name": "aws_storage",
            "sane": true,
            "type": "S3",
            "warning": false
        },
        {
            "name": "fs_storage",
            "reason": "\"/tmp/\" is readable and writable",
            "sane": true,
            "type": "filesystem",
            "warning": false
        }
    ],
    "volumes": [
        {
            "name": "bob/aws_volume",
            "sane": true,
            "warning": false
        },
        {
            "name": "bob/fs_volume",
            "sane": true,
            "warning": false
        }
    ]
}
