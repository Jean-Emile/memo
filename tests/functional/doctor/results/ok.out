[OK] User
[OK] Silos
[OK] Networks
[OK] Volumes
[OK] Drives
[OK] Leftovers
